Date.js

```diff
@@ -43,3 +43,25 @@ function getDayOfWeekInShortStringFormat(date, locale) {
 function getHour(date) {
   return (new Date(date)).getHours(); // Returns the hour (from 0-23)
 }
+
+function getTheDate(date) {
+  return new Date(date).getDate();
```

Comment:
name this function `getDate`

Reply:
getDate() is a predefined function, that is why I named this one getTheDate.

Reply:
where are you using this method? I would like this to be renamed to something else entirely to avoid confusion. Could you please point me to the place where this is used? and what is the expected type of the `date` parameter? is it a string/number/etc ?